src/main/java/uk/gov/justice/digital/delius/service/CustodyService.java

```diff
@@ -228,18 +228,22 @@ public Custody getCustodyByConvictionId(final String crn, final Long convictionI
     }
 
     @Transactional
-    public Custody offenderRecalled(final String nomsNumber, final LocalDate occurred) {
+    public Custody offenderRecalled(final String nomsNumber, final OffenderRecalledNotification recalledNotification) {
         final var offender = offenderRepository.findByNomsNumber(nomsNumber)
             .orElseThrow(() -> new NotFoundException(String.format("Offender with nomsNumber %s not found", nomsNumber)));
 
-        log.info("Offender {} recalled on {}", nomsNumber, occurred);
+        log.info("Offender {} recalled from {} on {}", nomsNumber, recalledNotification.getNomsPrisonInstitutionCode(), recalledNotification.getRecallDate());
+        final var telemetryProperties = Map.of("offenderNo", nomsNumber,
+            "recallDate", recalledNotification.getRecallDate().format(DateTimeFormatter.ISO_DATE),
+            "institution", recalledNotification.getNomsPrisonInstitutionCode());
 
         try {
             Event event = convictionService.getActiveCustodialEvent(offender.getOffenderId());
-            telemetryClient.trackEvent( "P2POffenderRecalled");
+
+            telemetryClient.trackEvent("P2POffenderRecalled", telemetryProperties, null);
```

Comment:
not sure if this telemetry looks correct - I thought prison to probation used `P2P` rather than community api?

Reply:
I kinda followed the pattern that was already being used.  it uses the P2P prefix for those services currently called by P2P.